tests/test_execution.py

```diff
@@ -1276,3 +1276,43 @@ def test_inheritance_direct_method_invocation(harness: _TestHarness) -> None:
         "ChildContract.method called",
         "GrandParentContract.method called",
     ]
+
+
+def test_account_from_bytes_validation(harness: _TestHarness) -> None:
+    def test() -> None:
```

Comment:
I thought we stopped doing tests from closures?

Reply:
Why's that?

Reply:
You removed them previously (I forget why), they also are a bit slower since they aren't part of the AWST caching

Reply:
I don't recall doing that 🤔

Anyway, these ones are here to avoid too much cruft in `test_cases`, since they're basically one liners.